### DISCUSSION TITLE
Choppy Video Playback

### DISCUSSION THREAD

**Discussion by fhrivers** [AntonyCorbett/OnlyM#457] [Video Problems]
I'm getting slightly choppy MP4 playback with OnlyM. The files are around 300 MB in size. The choppiness is slight but noticeable during playback and got progressively worse near the end. Audio started cutting out. I'm also sharing the screen via Zoom.

I played a second video in the same series using the video player that comes pre-installed in Windows 11 and it was flawless so it's not a hardware issue, I believe.

System Specs:

Surface Pro 8
Core i7 11th Gen
16 Go RAM
Windows 11 Home

Are there any settings I need to tweak?

**Comment by scottys94** [comment]
@fhrivers Which rendering engine is selected in your OnlyM settings?

**Reply by fhrivers** [reply]
Media Foundation

**Reply by scottys94** [reply]
@fhrivers We had similar issues in the past, and changing the rendering engine to Ffmpeg resolved it for us.
